### PR TITLE
Demobranchdocs

### PR DESCRIPTION
**Pull request by rmdash**
Turned "DGBB" to "DAO"

Preview: https://dash-docs--140.org.readthedocs.build/en/140/governance/index.html

### CODE REVIEW

**Comment by thephez** [comment]
Thanks 👍 Could you update other pages that use that term as well (https://docs.dash.org/en/stable/search.html?q=dgbb&check_keywords=yes&area=default)? Also, the keywords section at the top of some pages (e.g. governance/index.rst) includes it. We can add "dao" to that list.

File: governance/index.rst [hunk 1]
```diff
@@ -8,34 +8,11 @@
...
+
+The DAO also provides a means for Dash to fund its own development. While other projects have to depend on donations or premined endowments, Dash uses 10% of the block reward to fund its own development. Every time a block is mined, 90% of the reward is split between the miner and a masternode per the distribution found :ref:`here <block-reward-reallocation>`, while the remaining 10% is not created until the end of the month. During the month, anybody can make a budget proposal to the network. If that proposal earns the net approval of at least 10% of the masternode network, then at the end of the month the requested amount will be paid out in a "superblock". At that time, the block rewards that were not paid out (10% of each block) will be used to fund approved proposals. The network thus funds itself by reserving 10% of the block reward for budget projects.
+
+In late 2016, IOHK prepared a detailed report on version 0.12.1 of the Dash governance system, including formal analysis of weaknesses and areas for improvement. You can view the report here. An updated overview of cryptocurrency treasury system in general was published by IOHK in 2019, available here.
```
Comment (thephez) [inline governance/index.rst:15]: Looks like this got duplicated from it's original spot below the video.
```suggestion
```

File: governance/index.rst [hunk 2]
```diff
@@ -8,34 +8,11 @@
...
-masternode network, then at the end of the month the requested amount
-will be paid out in a "superblock". At that time, the block rewards that
-were not paid out (10% of each block) will be used to fund approved
-proposals. The network thus funds itself by reserving 10% of the block
-reward for budget projects.
+Governance in a decentralized project is difficult, because by definition there are no central authorities to make decisions for the project. In Dash, such decisions are made by a Decentralized Autonomous Organization (or DAO) consisting of the masternodes. The DAO allows each masternode to vote once (yes/no/abstain) for each proposal. If a proposal passes, it can then be implemented (or not) by Dash's developers. A key example is early in 2016, when Dash's Core Team submitted a proposal to the network asking whether the blocksize should be increased to 2 MB. Within 24 hours, consensus had been reached to approve this change. Compare this to Bitcoin, where debate on the blocksize has been raging for nearly three years and has resulted in serious splits within the community and even forks to the Bitcoin blockchain.
```
Comment (thephez) [inline governance/index.rst:11]: Can you wrap the text similar to how it was before?

**Review by thephez — COMMENTED** [review]
:+1: Just a couple small things to update.

**Review by thephez — COMMENTED** [review]
A few more DGBB references that we missed. There's one in the keywords list of governance/understanding.rst and a few more in introduction/features.rst

File: governance/index.rst [hunk 3]
```diff
@@ -8,47 +8,44 @@
...
+created until the end of the month. During the month, anybody can make a budget 
+proposal to the network. If that proposal earns the net approval of at least 10% 
+of the masternode network, then at the end of the month the requested amount will 
+be paid out in a "superblock". At that time, the block rewards that were not paid 
+out (10% of each block) will be used to fund approved proposals. The network thus 
+funds itself by reserving 10% of the block reward for budget projects.
+
+In late 2016, IOHK prepared a detailed report on version 0.12.1 of the Dash governance 
+system, including formal analysis of weaknesses and areas for improvement. 
+You can view the report `here <https://iohk.io/research/papers/#NSJ554WR>`__.
```
Comment (thephez) [inline governance/index.rst:38]: Just checked this and it's a broken link. Seems that IOHK pulled the doc (https://iohk.io/en/research/library/papers/dash-governance-systemanalysis-and-suggestions-for-improvement/) so I guess we need to drop the first part of this paragraph and revise the end.

**Comment by rmdash** [comment]
MIght as well remove that entire last section since it isnt adding anything
by itself.

**Review by thephez — APPROVED** [review]
:+1:

**Pull request merged**